README.md

```diff
@@ -5,17 +5,14 @@ A python wrapper for resin.io [work in progress]
 ## usage 
 
 ```
-pip install resin
-```
```

Comment:
We should probably also mention that this is configured for resinstaging api incase any users come across this.